README.adoc

```diff
@@ -285,6 +285,104 @@ spec:
 ...
 ----
 
+== Configure Health
+
+If you want customize the health alerts in Kiali you can do it in the configuration yaml.
+
+=== Kiali defaults
+
+By default Kiali health is:
+
+* Degraded when Error Rate is >= 0.1%
+* Failure when Error Rate is >= 20%
+
+For Kiali an error request is when:
+
+* For http the code status match the expression */^[4-5]\d\d$/*
+* For grpc the code status match the expression */^ [1-9]$|^1[0-6]$/*
+
+=== Configuration
+
+Kiali works with a FIFO configuration, so Kiali will use the first configuration that applies with the resource requested.
+
+So the granularity and criteria should be configured from highest to lowest.
+
+The health configuration follow this scheme:
+
+[source,yaml]
+----
+health_config:
+  rate:
+    - namespace: string
+      kind: string
+      name: string
+      tolerance:
+        - request_protocol: string
+          request_type: string
+          code: string
+          degraded: number
+          failure: number
+          ...
+    ...
+
+
+----
+
+For each Rate Kiali will use the first match by:
+
+* Namespace (string) is a regular expression that match the namespace name (bookinfo, istio-system...)
+* Kind (Optional) (string) is a regular expression that match the resource type (app/workload/service)
+* Name (Optional) (string) is a regular expression that match the resource name (reviews, productpage...)
+
+If a configuration match kiali will use the first tolerance configuration that match:
+
+* Request protocol is a plain string (http/grpc...)
+* Request type (string) is a regular expression that match the request (inbound/outbound)
+* Code (string) is a regular expression that match for each request the status_code to calculate the Error Rate.
+* Degraded is the threshold to show a *"degraded"* state for a resource
+* Failure ios the threshold to show a *"failure"* state for a resource
+
+
+Example configuration:
+
+[source,yaml]
+----
+health_config:
+  rate:
+    - namespace: "default"
+      tolerance:
+        - request_protocol: "http"
+          request_type: "bound"
+          code: "400"
+          degraded: 5
+          failure: 10
+        - request_protocol: "grpc"
+          request_type: "inbound"
+          code: "^1$"
+          degraded: 10
+          failure: 15
+    - namespace: "bookinfo"
+      kind: app
+      name: reviews
+      tolerance:
+        - request_protocol: "http"
+          request_type: "outbound"
+          code: "400"
+          degraded: 1
+          failure: 5
+----
+
+In this case we created :
+
+*  For namespace default:
+  ** A "degraded" alert in http protocol and if request type match the regular expression bound (In this case for inbound and outbound), when code mach the regular expression 400 rate is >=5% and failure alert if >=10%
```

Comment:
It's not an alert here ... it's actually a threshold (I know Lucas didn't want to use this word, but "alert" is even worse, no?)